antlr-kotlin-runtime/src/commonMain/kotlin/org/antlr/v4/kotlinruntime/ParserRuleContext.kt

```diff
@@ -191,34 +194,6 @@ public open class ParserRuleContext : RuleContext {
     return addAnyChild(errorNode)
   }
 
-  /**
-   * Add a child to this node based upon [matchedToken].
-   *
-   * It creates a [TerminalNodeImpl] rather than using [Parser.createTerminalNode].
-   * I'm leaving this in for compatibility but the parser doesn't use this anymore.
-   */
-  @Deprecated("Not used anymore")
-  public fun addChild(matchedToken: Token): TerminalNode {
```

Comment:
Could this be useful to someone wanting to modify the tree directly (i.e., not through the parser)?

Reply:
You can still pass in a `TerminalNode` with the other method `addChild(t: TerminalNode)`, the only difference is you'll have to construct a `TerminalNode` youself, instead of relying on the runtime.